### ISSUE TITLE
Change Blogger default theme :)

### ISSUE THREAD

**Issue by autumnfjeld** [Automattic/wp-calypso#55277]
# Description
This is part of [iteration #2 Write & Build flow ]( pdgK6S-78-p2#write-build-intent-screen in the  production of Blogger Flow v1). 👏

The existing WP.com themes don't serve bloggers well.
"Let's get rid of the blue trees." - @jordesign ;)

We want users to get to writing as soon as possible, and one way to encourage that is to put a simple, elegant, easy-to-blog-with design in front of them as soon as they land in the editor.  Project thread for this: pdgK6S-1h-p2

This will be a single post, query block (aka Block List block) on a static page.
Introducing Zoologist: p7DVsv-cSH-p2

**BEFORE:** Current default Hever theme

![image](https://user-images.githubusercontent.com/5665959/128664769-8798f839-2402-4c28-b646-c47dd03f9fcc.png)

**AFTER:** with [Zoologist](Pdqn8m-2-p2)

![zoologist_new](https://user-images.githubusercontent.com/4550351/137648379-0479315b-c72f-440c-9f1b-f3217a0d4c76.jpeg)

# Dev Tasks
- [x] #57134
- [x] Translate annotations and deploy D69509-code
- [x] Be sure to alert @jordesign  before shipping so he can share with HE.

## Links / References
Comments about query block not being available pcjTuq-lX-p2

**Comment by alaczek** [comment]
I also wanted to include this summary:
"Let's get rid of the blue trees." - @jordesign ;)

We also have a project thread for this: pdgK6S-1h-p2

**Comment by jordesign** [comment]
I've removed the project thread - so we can just use this issue.

**Comment by autumnfjeld** [comment]
It's been decided! We'll use the the Zoologist theme (which is a derivative of the Geologist theme dmli5-p2)
P2: pNEWy-ekS-p2#comment-53631

**Comment by autumnfjeld** [comment]
@niranjan-uma-shankar  I was told you are wise in the ways of setting the default theme for Wordpress.  We are changing the default for WP.com from Hever to Zoologist.  (See description above)

@roo2 or @arthur791004 may need some input when they are ready to do this--needed for launching the new Blogger Flow.

**Comment by jordesign** [comment]
If we can give Happiness a 24 hour heads up - I think that will be sufficient. They already know the theme is launched - but so we can give a shorter turnaround on it becoming the default

**Comment by niranjan-uma-shankar** [comment]
> @roo2 or @arthur791004 may need some input when they are ready to do this--needed for launching the new Blogger Flow.

Happy to help share inputs or in any other way. Please let me know how I can contribute.

**Comment by ianstewart** [comment]
@autumnfjeld do we need a separate issue to make this the default for everyone? Not just Blogger flow? Also, here's a screenshot since I have one handy from @alaczek. :)

![image](https://user-images.githubusercontent.com/545779/137823857-5b0a6279-ad2e-4e00-8aca-f231b8cfd499.png)

**Comment by p-jackson** [comment]
https://user-images.githubusercontent.com/1500769/137841784-f218ae0a-109c-4e1e-8f42-93529e412840.mp4

Got a PR up for changing the default theme

> do we need a separate issue to make this the default for everyone? Not just Blogger flow?

Not necessary, this PR changes it everywhere.

This change doesn't impact the `/new` flow since there's no skip button for the design picker. The user _has_ to select a design so there's no default to change.

This minimal starter content hasn't been translated, which means that non-en users will get English starter content. If we're happy with this starter content then I can get the process gears turning so that it can all be translated before merging.

**Comment by p-jackson** [comment]
Default has been updated by #57134
